metrics-core/src/main/java/com/codahale/metrics/SharedMetricRegistries.java

```diff
@@ -41,4 +43,20 @@ public static MetricRegistry getOrCreate(String name) {
         }
         return existing;
     }
+
+    public synchronized static MetricRegistry setDefault(String name) {
```

Comment:
Should there be another method for instances when you already have the MetricRegistry? I am using Dropwizard where a metrics registry is created by the framework and I would like to make that instance the default.

As proposed:
```java
SharedMetricRegistries.add("default", bootstrap.getMetricRegistry());
SharedMetricRegistries.setDefault("default");
```

With the overloaded setDefault method.
```java
SharedMetricRegistries.setDefault("default", bootstrap.getMetricRegistry());
```